astropy/units/format/ogip.py

```diff
@@ -24,6 +24,7 @@
 from fractions import Fraction
 from typing import TYPE_CHECKING
 
+from astropy.units.errors import UnitsWarning
```

Comment:
Same as fits: import location and perhaps also import `CompositeUnit` to get rid of `core`